stripe/test/resources/test_country_specs.py

```diff
@@ -0,0 +1,23 @@
+import stripe
+from stripe.test.helper import StripeResourceTest
+
+
+class CountrySpecTest(StripeResourceTest):
+
+    def test_retrieve_country_spec(self):
```

Comment:
Mind just doing two minor tweaks here?

1. Re-order the naming to look like our other specs in the form of `test_<resource>_<action>`. Actually, see [the competing pull for an example](https://github.com/stripe/stripe-python/pull/209/files) :)
2. Put `list_all` in above `retrieve`. This seems to be more consistent with other suites.